index.html

```diff
@@ -184,11 +184,13 @@ <h2 class="marketing-strategies__title">Marketing Strategies</h2>
       <footer class="footer">
```

Comment:
Use semantic tags for lists. (Apply for .points)

Reply:
completed